src/ui/page/course/video.html

```diff
@@ -7,15 +7,15 @@ <h1 class="heading-xlarge heading heading--page-heading ">
             <div class="column-two-thirds">
                 <div class="aspect__container">
                     <div class="video aspect__content">
-						{#if !module.url.endsWith('.mp4') }
+						{#if module.url.startsWith('https://') }
```

Comment:
Have a review of this logic, not sure checking if module url starts with 'https://' is the best way of determining whether youtube or mp4